hybridauth/callbacks/linkedin2.php

```diff
@@ -0,0 +1,11 @@
+<?php
+/*!
+* WordPress Social Login
+*
+* https://miled.github.io/wordpress-social-login/ | https://github.com/miled/wordpress-social-login
+*   (c) 2011-2019 Mohamed Mrassi and contributors | https://wordpress.org/plugins/wordpress-social-login/
+*/
+
+$provider_id = "LinkedIn2";
+
+require "../callback.php";
```

Comment:
You must add **?>** To the end of the file

Reply:
None of the files in the callbacks folder ends in ?> so I kept the project folder format